helpers/sample_utils.ts

```diff
@@ -22,8 +25,12 @@ export function fakeDefinitionToMetadata(def: FakePackDefinition): PackMetadata
     ...packMetadata
   } = def;
   const formulas = originalFormulas!.map(formula => {
-    const {execute, ...formulaMetadata} = formula;
-    return formulaMetadata;
+    const {execute, validateParameters, ...formulaMetadata} = formula;
+    const validateParametersAsFormula = wrapMetadataFunction<ExecutionContext, boolean>(validateParameters);
```

Comment:
How come we never needed to use wrapMetadataFunction here for the existing metadata formulas? Is it just because this file kinda ignores sync tables or is sloppy about them?

Reply:
We handle getConnectionName by stripping out its execute attribute. For the sync table metadata formulas, the input type for sync tables doesn't seem to accept a MetadataFunction, only MetadataFormulas already